pyresample/test/test_sgeom/test_arc.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2022 Pyresample developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Define tests for the SArc class."""
+
+import copy
+import unittest
+
+import numpy as np
+import pytest
+from shapely.geometry import LineString
+
+from pyresample.future.spherical import SArc, SPoint
+
+
+class TestSArc(unittest.TestCase):
```

Comment:
Looks like there is no need for this class (a new one) to be based on unittest.

```suggestion
class TestSArc:
```